apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.ts

```diff
@@ -29,4 +31,19 @@ router.get(
   }),
 );
 
+router.get('/generation_logs.json', asyncHandler(async (req, res) => {
```

Comment:
For a logging route that only I'm going to use, should I add a download button to the UI (for ease of access), or leave it as a route without a link (to keep the UI clean)?

Reply:
I think it's fine to omit this from the UI.